towerlib/entities/job.py

```diff
@@ -715,6 +783,18 @@ def _get_dynamic_value(self, variable):
         response = self._tower.session.get(url)
         return response.json().get(variable) if response.ok else None
 
+    @property
+    def workflow_nodes(self):
+        """Workflow nodes executed in the workflow.
+
+        Returns:
+          JSON Results on success, None otherwise.
+
+        """
+        url = f'{self._tower.api}/workflow_jobs/{self.id}/workflow_nodes'
+        response = self._tower.session.get(url)
+        return response.json().get('results', {}) if response.ok else None
```

Comment:
Wouldn't it a bit more concistent to the type here to return an empty dictionary on failure instead of None?

Reply:
Yeah it might be. I did try to gauge how you were handling your return types, so happy to change this to an empty dictionary